### PR TITLE
Position algo

### PR DESCRIPTION
**Pull request by nibess1**
theoretically everything should work, it works on test, performed integration tests with the db, but haven't done controller tests with frontend. Theres also no frontend.

### CODE REVIEW

File: backend/src/main/java/com/project/G1_T3/player/controller/PlayerProfileController.java [hunk 1]
```diff
@@ -23,7 +23,7 @@ public class PlayerProfileController {
     @PostMapping("/create") // Map to URL for creating player profile
     @ResponseStatus(HttpStatus.CREATED)
     public PlayerProfile create(@RequestBody PlayerProfile playerProfile) {
```
Comment (Xskullibur) [inline backend/src/main/java/com/project/G1_T3/player/controller/PlayerProfileController.java:25]: Maybe use a more meaningful method name and don't use verbs as the mapping unless it's really necessary

File: backend/src/main/java/com/project/G1_T3/leaderboard/model/LeaderboardPlayerProfile.java [hunk 2]
```diff
@@ -18,21 +18,21 @@ public class LeaderboardPlayerProfile {
     UUID profileId;
     String firstName;
     String lastName;
-    float ELO;
+    int glickoRating;
     Long position;
 
-    public LeaderboardPlayerProfile(UUID profileId, String firstName, String lastName, Float eLO) {
+    public LeaderboardPlayerProfile(UUID profileId, String firstName, String lastName, int glickoRating) {
         this.profileId = profileId;
```
Comment (ceewaigit) [inline backend/src/main/java/com/project/G1_T3/leaderboard/model/LeaderboardPlayerProfile.java:25]: why the spelling of eLO like that HAHAHA

File: backend/src/main/java/com/project/G1_T3/player/controller/PlayerProfileController.java [hunk 3]
```diff
@@ -48,16 +48,16 @@ public ResponseEntity<PlayerProfile> getUserByPlayerId(@PathVariable String id)
...
+    public ResponseEntity<Double> getPercentilePlayerRankByUserId(@PathVariable String userId) {
         PlayerProfile playerProfile = playerProfileService.findByUserId(userId);
         
         if (playerProfile == null) {
             return ResponseEntity.notFound().build();
         }
         
         // Get the rank (position) of the player by their profileId
-        int rank = playerProfileService.getPlayerRank(playerProfile.getProfileId().toString());
+        Double rank = playerProfileService.getPlayerRank(playerProfile.getProfileId().toString());
```
Comment (ceewaigit) [inline backend/src/main/java/com/project/G1_T3/player/controller/PlayerProfileController.java:60]: why is this a double

Comment (ceewaigit) [reply backend/src/main/java/com/project/G1_T3/player/controller/PlayerProfileController.java:60]: oh rank percentage

**Review by ceewaigit — CHANGES_REQUESTED** [review]
just the eLO thing AHHAHAH

File: backend/src/main/java/com/project/G1_T3/leaderboard/model/LeaderboardPlayerProfile.java (hunk 2 above)
Comment (nibess1) [reply backend/src/main/java/com/project/G1_T3/leaderboard/model/LeaderboardPlayerProfile.java:25]: wait thanks for changing it to glickoRating though.

**Comment by nibess1** [comment]
i checked again and there isnt anthing like eLO anymore. LGTM

**Review by ceewaigit — APPROVED** [review]

**Pull request merged**